### ISSUE TITLE
[QUESTION] Questions about supporting Windows version of Worker

### ISSUE THREAD

**Issue by xiashao1980** [apache/incubator-dolphinscheduler#1335]
Hello,
Our business is mainly media processing, so many servers are Windows operating systems. We want to develop some workers to use these Windows servers. Is it possible? In addition, because different servers are in the network or the role of the server itself, workers on different servers will process different services. How is this distinguished in the implementation of workers? For example, the server we transcoded cannot be used for file migration and the like.
To summarize, my questions are:
1. Is it easy to implement a worker for Windows?
2. Can I assign different jobs to different types of workers?

**Comment by dailidong** [comment]
1、it is not  very difficult to implement it， but will cost some time to test

2、for now,  When you configure a job, you can set which worker groups the job is executed on,  it has been supported. so this question is not a problem.

by the way, you should split this question to 2 issues,  and please update/add title/description in English, thx!

if you need help, please add my wechat:510570367,  when added, please let me know you want to implement "supoort windows feature"

best regards

**Comment by CalvinKirs** [comment]
Because of the license, we cannot support it.
